src/components/Aframe/my-loader.js

```diff
@@ -314,13 +319,11 @@ AFRAME.registerComponent("myloader", {
       var data = this.data;
```

Comment:
is `data` used? I only see `this.data`s

Reply:
same with `el`

Reply:
yes, they are being used in the promise call